File: services/telegramservice/DTOs/DTO/NotificationsDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace DTOs.DTO
{
    public class NotificationsDTO
    {
        public long? Id { get; set; }
        public DateTime? Timestamp { get; set; }
        public string? Type { get; set; }
        public string? Message { get; set; }
        public bool? read { get; set; }
    }
}

File: services/prod/src/main/java/com/torin/prod/dto/NotificationsDto.java
package com.torin.prod.dto;

import java.time.Instant;


import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class NotificationsDto {
    private Long id;
    private Instant timestamp;
    private String type;
    private String message;
    private Boolean read;
    private String formattedDate;

    public NotificationsDto(Instant timestamp, String type, String message) {
        this.timestamp = timestamp;
        this.type = type;
        this.message = message;
        this.read = false;
    }
}
